SteamKit2/SteamKit2/Steam/Handlers/SteamUser/Callbacks.cs

```diff
@@ -296,18 +296,25 @@ public sealed class WalletInfoCallback : CallbackMsg
             /// Gets the currency code for this wallet.
             /// </summary>
             public ECurrencyCode Currency { get; private set; }
+
             /// <summary>
-            /// Gets the balance of the wallet, in cents.
+            /// Gets the balance of the wallet as a 32-bit integer, in cents.
```

Comment:
I know that stating data type in the doc is a bit excessive and unneeded, but I wanted to point out the difference between those two for people that might not recall at first. I think it's OK in this case where consumer might wonder which one to use.

Reply:
Do we know what actually happens to this field when an account has a balance of $42m?

Reply:
Is it in USD or users currency?

Because with vietnamese dong a game can cost 1.300.000₫ which is 130000000 cents.

Reply:
Well do we know what happens in that scenario too?

Reply:
No clue, but I pointed it out to ease checking it out if anyone wants to.

Reply:
@JustArchi Just saw your comment - yeah, but it's Valve. I'd be quite unsurprised if it's just the lower 32 bits of the actual balance, and wraps around to the negatives.

Reply:
@yaakov-h you're totally right and it also wouldn't shock me, but I'm not rich enough to check 😀.

And BTW yes, this balance is in local currency (`ECurrencyCode`) which is user's local currency, if supported by Steam. It counts from full cents or other smallest monetary value.